procurify-dashboard/index.html

```diff
@@ -0,0 +1,317 @@
+            <a href="">
+              <span class="pending-number">2</span>
+              <p class="pending-title">Travel Pending Approval</p>
+            </a>
+        </div>
+      </div>
+
+      <div class="row active-row">
+        <div class="col-7">
+          <div class="panel activity-panel">
+            <div class="activity-content">
+              <div class="panel-title">Activity</div>
+            </div>
+            <ul class="panel-navigation">
+              <li class="active">Orders</li>
+              <li>Expense Reports</li>
+              <li>Travel</li>
+            </ul>
+            <div class="activity-wrap">
+                  <div class="activity-heading user-details row">
+                      <div class="col-6">
+                          <a class="object">
+                            <div class="object-icon">
+                              <img src="http://abduzeedo.com/sites/default/files/originals/63576860118469.5a3c63a44da98.jpg" alt="">
+                            </div>
+                          </a>
+                          <div class="user-info">
+                            <h5>Sohn Jtamos</h5>
+                          </div>
+                        </div>
+                        <div class="user-info activity-order-info col">
+                      <p class="sub-info activity-date">Aug 10, 2018, 12:45 PM</p>
+                    </div>
+                    </div>
+                    <div class="activity-heading row">
+                    <div class="col">
+                    <div class="user-info activity-status-wrap">
+                      <p class="activity-status">Item Purchased</p>
+                      <p class="sub-info">from <span>Order #113</span></p>
+                    </div>
+                  </div>
+                    </div>
+                  <!-- <div class="activity">
+                    <p class="activity-full">Croissant danish candy canes bear claw dragée pie gingerbread. Dragée jelly beans jelly pudding bonbon. Chocolate apple pie toffee cookie bonbon oat cake halvah. Gummies marzipan muffin candy pastry chocolate bar.</p>
+                  </div> -->
+            </div>
+            <div class="activity-wrap">
+                  <div class="activity-heading user-details row">
+                      <div class="col-6">
+                          <a class="object">
+                            <div class="object-icon">
+                              <img src="http://abduzeedo.com/sites/default/files/originals/63576860118469.5a3c63a44da98.jpg" alt="">
+                            </div>
+                          </a>
+                          <div class="user-info">
+                            <h5>Sohn Jtamos</h5>
+                          </div>
+                        </div>
+                        <div class="user-info activity-order-info col">
+                      <p class="sub-info activity-date">Aug 10, 2018, 12:45 PM</p>
+                    </div>
```

Comment:
Watch out for indentation here and the following few lines. It makes it much harder to read which tags open and where they close!